pkg/utils/packet_test.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"time"
+
+	mocks_utils "github.com/k8snetworkplumbingwg/sriov-cni/pkg/utils/mocks"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
nit: maybe dot import ginkgo and gomega ?

Reply:
I personally wouldn't. I also see that there are no dot imports anywhere else. But I don't mind, if that's the preferred style... (?).

Reply:
it is preferred.

[1]https://github.com/k8snetworkplumbingwg/sriov-cni/blob/aacf339ed0c485c8edb22024e091a2720266c4cb/pkg/sriov/sriov_test.go#L13
[2] https://github.com/k8snetworkplumbingwg/sriov-cni/blob/aacf339ed0c485c8edb22024e091a2720266c4cb/pkg/utils/utils_test.go#L11

Reply:
ah, I grepped for `"import \.", and missed those uses... Will update.